firehose-source/src/main/java/source/FirehoseProperties.java

```diff
@@ -0,0 +1,118 @@
+/*
+ *  Copyright 2015 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package source;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Vinicius Carvalho
+ */
+@ConfigurationProperties(prefix = "firehose")
```

Comment:
@ericbottard thanks for the comprehensive review of this PR man, this is the only point I'll actually make a point in favor of this use. I think that if one app has more than one module (maybe that never happens), but when accessing the /env endpoint for example, I'd like to see some namespace qualification of where some properties come from. Just my thought on the subject, I'm not part of the team, and its up to you guys to decide the best practices, but I really would like to see prefix on configuration for ease management later.